src/Bridge/Doctrine/Orm/AbstractPaginator.php

```diff
@@ -41,6 +41,11 @@ public function __construct(DoctrinePaginator $paginator)
         $this->maxResults = $maxResults;
     }
 
+    public function getQuery(): Query
```

Comment:
This can actually be a good idea, although we need this in the interface we're implementing. To avoid a BC break you need to create a new interface `QueryAwarePartialPaginator` (that extends the current `PartialPaginatorInterface`).